src/test/java/com/iota/iri/service/APIIntegrationTests.java

```diff
@@ -200,6 +207,26 @@ public void shouldTestGetNodeInfo() {
             body(containsString("transactionsToRequest"));
     }
 
+    @Test
+    public void shouldTestGetIotaConfig() {
+
+        final Map<String, Object> request = new HashMap<>();
+        request.put("command", "getNodeAPIConfiguration");
+
+        given().
+            body(gson().toJson(request)).
+            when().
+            post("/").
+            then().
+            spec(specSuccessResponse).
+            body(containsString("maxFindTransactions")).
+            body(containsString("maxRequestList")).
+            body(containsString("maxGetTrytes")).
+            body(containsString("maxBodyLength")).
+            body(containsString("testNet")).
+            body(containsString("milestoneStartIndex"));
+    }
+
```

Comment:
Great test,

You should know that we now have `python-regression` tests using [aloe](https://aloe.readthedocs.io/en/latest/).
We are now almost completing our transition period and we may do away with the java integration tests.

You are not expected to change the aloe tests yet since we didn't give instructions on how to easily run them.
I opened https://github.com/iotaledger/iri/issues/1284 now.
@DyrellC when this PR is merged can you fix the relevant test?

Reply:
@GalRogozinski can do